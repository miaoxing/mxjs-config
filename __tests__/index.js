import {ConfigProvider, useConfig} from '../index';
import {render} from '@testing-library/react';

describe('config', () => {
  test('basic', () => {
    const Child = () => {
      const config = useConfig();
      return config.key;
    };

    const {container} = render(<ConfigProvider config={{
      key: 'value',
    }}>
      <Child/>
    </ConfigProvider>);
    expect(container).toMatchSnapshot();
  });
});
