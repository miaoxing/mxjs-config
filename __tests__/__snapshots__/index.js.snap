// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config basic 1`] = `
<div>
  value
</div>
`;
